packages/app/hooks/useChannelNavigation.old.ts

```diff
@@ -0,0 +1,76 @@
+import { isGroupChannelId } from '@tloncorp/shared/dist';
```

Comment:
If we're not actually referencing this file anyway, I think it's fine to discard